tools/ansible/tasks/logging/test/logging_test.go

```diff
@@ -1,18 +1,26 @@
 package test
 
 import (
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 	"os/exec"
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
+// Pining down version of Ruby Devkit to 2.5, 64 bit version
+// Ruby installation path kept default C:\
+var rubyExecutablePath = "C:\\Ruby25-x64\\bin\\ruby.exe"
```

Comment:
Why is this needed?

Reply:
os.exec for some reason isn't able to map `ruby` to the executable path of Ruby (In this case it is C:\Ruby25-x64\bin\ruby.exe), even though the bin path has been added to `PATH` environment variable as a part of installation.
os.exec only works for fully qualified path in case of `ruby --version`, otherwise it throws file-not-found error. So the fully qualified path is explicitly mentioned here.

Reply:
That is because you are not running it in a shell like cmd or powershell. Only then will the PATH env get picked up.

Reply:
I'm not sure that is the only reason, considering the command `exec.Command("fluent-gem", "list").Output()` in `func TestPluginsInstalled` seems to be working but `exec.Command("ruby", "--version").Output()` fails.
The `PATH` variable on the instance only reflects the bin path of Ruby, not the fluent-gem.
The error for `ruby --version` is exit status 2, which according to https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499- is `ERROR_FILE_NOT_FOUND` or `The system cannot find the file specified.`, but stdout and stderr are returned empty.
Consequently I propose to keep the executable path as fully qualified path for the unit test

Reply:
This sounds very strange and needs further digging into maybe on a hack day